map-widget/qml/MapWidget.qrc

```diff
@@ -0,0 +1,27 @@
+<RCC>
+	<qresource prefix="/">
+		<file>MapWidget.qml</file>
+		<file>MapWidgetContextMenu.qml</file>
+		<file>MapWidgetError.qml</file>
+
+		<file alias="dive-location-marker-icon">icons/mapwidget-marker.png</file>
+		<file alias="dive-location-marker-inactive-icon">icons/mapwidget-marker-gray.png</file>
+		<file alias="dive-location-marker-selected-icon">icons/mapwidget-marker-selected.png</file>
+		<file alias="map-style-map-icon">icons/mapwidget-toggle-street.png</file>
+		<file alias="map-style-photo-icon">icons/mapwidget-toggle-satellite.png</file>
+		<file alias="open-menu-icon">icons/mapwidget-context-menu.png</file>
+		<file alias="zoom-in-icon">icons/mapwidget-zoom-in.png</file>
+		<file alias="zoom-out-icon">icons/mapwidget-zoom-out.png</file>
+
+		<!-- mobile
```

Comment:
mostly just curious - why is there this section of resources that are commented out?

Reply:
That was while I tested, let me remove that fast.